runtime/testnet/src/config/contracts.rs

```diff
@@ -64,7 +53,7 @@ impl pallet_contracts::Config for Runtime {
 	// if a too-large contract is uploaded. We noticed that it poses
 	// less friction during development when the requirement here is
 	// just more lax.
-	type MaxCodeLen = ConstU32<{ 256 * 1024 }>;
+	type MaxCodeLen = ConstU32<{ 123 * 1024 }>;
```

Comment:
where does this come from? Normally these values are aligned / power-of-two bytes. `123` is not.

Reply:
Changed it back